### PR TITLE
Update 02.2022.1 - Definitions

### PR DESCRIPTION
**Pull request by Epride**
Questions to consider:

Are we happy with the definition for event (A particular contest within a WCA Competition). Seems clunky and maybe using “disciplines” could be more descriptive.

Do we want to split off the definition of Regional Organization into Associate and Full or just leave as one for now?

Do we want to add that a Senior Delegate is responsible for managing a group of Regional Delegates as well

Do we need a definition for the WCRP now?

### CODE REVIEW

File: documents/motions/02.2022.1 - Definitions.md [hunk 1]
```diff
@@ -52,15 +52,11 @@ Within WCA we use the following definitions for terminology.
...
-      <td>The Community of the WCA as formed by all Staff, Registered Speedcubers, and Partners.</td>
+      <td>The Community of the WCA as formed by all WCA Staff, Registered Speedcubers, and Partners.</td>
     </tr>
     <tr>
       <td>Competition Organizer</td>
-      <td>Persons who are responsible for the organization of a WCA Competition, as defined in the WCA Regulations.</td>
-    </tr>
-    <tr>
-      <td>Competition Staff</td>
-      <td>Persons who are responsible for the operational procedures during a WCA Competition, as defined in the WCA Regulations.</td>
```
Comment (Epride) [inline documents/motions/02.2022.1 - Definitions.md:63]: If we wanted to keep this role in the definitions we can but would need to change it to "Officials" or "Competition Officials" to match the wording of regulations article 1

**Comment by EdHollingdale** [comment]
In addition to the split of ROs into full and associate, should the motions also define sub-RO (currently this term is defined onlh in the RO support policy).

**Comment by Epride** [comment]
> In addition to the split of ROs into full and associate, should the motions also define sub-RO (currently this term is defined onlh in the RO support policy).

We do have 1.2 of the RO motion (6.2022.1) which permits sub RO's however if you think it is valuable we can add a definition to the definitions motion as well

**Comment by EdHollingdale** [comment]
> We do have 1.2 of the RO motion (6.2022.1) which permits sub RO's however if you think it is valuable we can add a definition to the definitions motion as well

I know we have 1.2 of the RO motion which establishes that sub-ROs can exist, I'm more talking about the term itself. Maybe just defining it in the RO support policy is sufficient.

**Comment by EdHollingdale** [comment]
To your questions Ethan, my take is:
- Definition of "Event" isn't currently causing issues and isn't "wrong", so suggest it remain unchanged
- If you split RO, as mentioned, I think it is worth formally defining sub-RO
- I don't think it is necessary to update the Senior Delegate definition. It is clear Regional Delegates report to Senior Delegates, and the "manage a group of WCA Delegates" necessarily includes Regional Delegates. 
- I think it is sufficient that "Policy" is defined in the motions, so adding specific policies is not needed

**Comment by EdHollingdale** [comment]
Should we add a definition for "WCA Document" given this is used in the Documents motion (and says they require Board approval). My view is that for the purposes of that motion it is just the other files listed in the "Documents" on the WCA website, and it makes sense those other documents (CoC, CoE, Vision and Strategy) would need WCA Board approval. I don't think other WCA documents stored elsewhere (organiser guidelines etc) need Board approval.

**Comment by Epride** [comment]
> Should we add a definition for "WCA Document" given this is used in the Documents motion (and says they require Board approval). My view is that for the purposes of that motion it is just the other files listed in the "Documents" on the WCA website, and it makes sense those other documents (CoC, CoE, Vision and Strategy) would need WCA Board approval. I don't think other WCA documents stored elsewhere (organiser guidelines etc) need Board approval.

Yeah I think this would be a good definition to add to clarify the term given it is used in a few places. Can also help in places where all the documents are listed out by replacing it with the word WCA Documents

**Comment by EdHollingdale** [comment]
> Yeah I think this would be a good definition to add to clarify the term given it is used in a few places. Can also help in places where all the documents are listed out by replacing it with the word WCA Documents

I've ended up just updating the WCA Documents Motion (see #31 ) to define it. I'm not sure it strictly needs to go in the definitions as well.

File: documents/motions/02.2022.1 - Definitions.md [hunk 2]
```diff
@@ -84,7 +80,7 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Country</td>
-      <td>A self-governing geographical area of the world recognized as an independent state by international law and international governmental bodies. <br> The Regulations contain the accepted list of countries.</td>
+      <td>Any country, state, territory or a part of territory recognised by the international community, in conformity with the Olympic Charter and the IOC regulations. <br> The Regulations contain the accepted list of countries.</td>
```
Comment (Nevseros) [inline documents/motions/02.2022.1 - Definitions.md]:
```suggestion
      <td>Any country, state, territory or a part of territory recognized by the international community, in conformity with the Olympic Charter and the IOC regulations. <br> The Regulations contain the accepted list of countries.</td>
```

Correct me if I'm wrong, but US English tends to use "z" in similar words.

Comment (EdHollingdale) [reply documents/motions/02.2022.1 - Definitions.md]: Ah yes, the danger of Australians writing in American English is that we know how to spell "correctly".

**Pull request closed without merging**